src/components/whiteframe/whiteframe.ts

```diff
@@ -0,0 +1,39 @@
+import {Component, Input, ElementRef,ViewEncapsulation, OnInit} from 'angular2/core';
+import {CONST} from 'angular2/src/facade/lang';
+
+@Component({
+    selector: '[md-whiteframe]',
+    templateUrl: './components/whiteframe/whiteframe.html',
+    styleUrls: ['./components/whiteframe/whiteframe.css'],
+    encapsulation: ViewEncapsulation.None
+})
+export class MdWhiteframe implements OnInit {
+    @CONST() static MIN_DP:number = 1;
+    @CONST() static MAX_DP:number = 24;
+    @CONST() static DEFAULT_DP:number = 4;
```

Comment:
How about `private elevation_:number;` ?

(I'm glad to get all the feedback. I've never contributed any project, so it's very new for me :smile: )